onnxruntime/core/mlas/lib/scalar/SgemmKernelScalar.cpp

```diff
@@ -144,10 +145,10 @@ Return Value:
                 Row1Block03 = Row1Block03 + BElements03 * Row1AElements0;
             }
 
-            BElements00 = B[4];
-            BElements01 = B[5];
-            BElements02 = B[6];
-            BElements03 = B[7];
+            BElements00 = b[16];
```

Comment:
in scalar, the packing wide is 4 not 16.

Reply:
Sorry, approval for CLA took a bit of time. Can this review be resumed?

As for message, I'll repeat that for all test cases both on x86_64 (when force-disabled optimized functions and generic are used) and s390x, I've seen only 16-elements packing.